exercises/concept/secrets/.docs/introduction.md.tpl

```diff
@@ -0,0 +1 @@
+%{concept: bitwise-operations}
```

Comment:
I think you also need a title here, that's not automated (but you can make sure by generating it)
```suggestion
# Introduction

%{concept: bitwise-operations}
```